tajo-common/src/main/java/org/apache/tajo/datum/DatumFactory.java

```diff
@@ -333,7 +330,7 @@ public static DateDatum createDate(Datum datum) {
     case DATE:
       return (DateDatum) datum;
     default:
-      throw new InvalidCastException(datum.type(), Type.DATE);
+      throw new TajoRuntimeException(Errors.ResultCode.INVALID_CAST, datum.type().toString(), Type.DATE.name());
```

Comment:
@hyunsik 
InvalidCastException is not subclass of TajoException, thus it can be used.
Should it be changed?